src/features/ModelPlan/ModelToOperations/_components/MatrixTable/columns.tsx

```diff
@@ -237,17 +259,71 @@ export const columns: ColumnType[] = [
       });
       return copyData;
     },
-    Cell: ({ row, rowType, expanded }: RowProps) => {
+    Cell: ({
+      row,
+      rowType,
+      expanded,
+      clearMessage,
+      setMTOModalOpen,
+      setMTOModalState,
+      initLocation,
+      search
+    }: ExtendedRowProps) => {
       if (rowType !== 'milestone') return <></>;
-      if (row.solutions.length === 0)
+      if (isMilestoneType(row)) {
+        if (row.solutions.length === 0)
+          return (
+            <>
+              <Button
+                type="button"
+                className="display-block"
+                unstyled
+                onClick={() => {
+                  if (clearMessage) clearMessage();
+                  if (setMTOModalState)
+                    setMTOModalState({
+                      modalType: 'selectSolution',
+                      milestoneID: row.id
+                    });
+                  if (setMTOModalOpen) setMTOModalOpen(true);
+                }}
+              >
+                {i18next.t('modelToOperationsMisc:table.selectASolution')}
+                <Icon.ArrowForward className="top-05 margin-left-05" />
+              </Button>
+            </>
+          );
+
         return (
-          <UswdsReactLink to="#">
-            Select a solution{' '}
-            <Icon.ArrowForward className="top-05 margin-left-05" />
-          </UswdsReactLink>
-        );
+          <>
+            {row.solutions.map((solution, index) => {
+              const solutionMap = findSolutionByKey(
+                solution.key!,
+                helpSolutions
+              );
 
-      return <>{row.solutions.join(', ')}</>;
+              const detailRoute = solutionMap?.route
+                ? `${initLocation}${search}${
+                    search ? '&' : '?'
+                  }solution=${solutionMap?.route || ''}&section=about`
+                : `${initLocation}${search}`;
+              return (
+                <React.Fragment key={solution.id}>
+                  {solution.key !== null ? (
+                    <UswdsReactLink to={detailRoute}>
+                      {solution.key}
+                    </UswdsReactLink>
```

Comment:
It looks like you're using the raw solution keys here, with underscores, to render the solution acronym.  The acronym should be used here is available.  Check with Zoe for solutions without acronyms, but should probably default to full name.
![Screenshot 2025-01-14 at 11 04 54 AM](https://github.com/user-attachments/assets/654a0cc7-547e-4053-9493-ff4547eb5567)